sql/core/src/test/resources/sql-tests/results/ansi/group-analytics.sql.out

```diff
@@ -1067,3 +1067,227 @@ struct<a:int,b:int,count(1):bigint>
 3	NULL	2
 3	NULL	2
 3	NULL	2
+
+
+-- !query
+SELECT a, b, count(1) FROM testData GROUP BY a, GROUPING SETS(ROLLUP(a, b))
```

Comment:
NOTE: Just in case, I've checked all the results here are the same with PostgreSQL ones.

Reply:
Thanks for your double check.